### DISCUSSION TITLE
multi-agent reinforcement learning

### DISCUSSION THREAD

**Discussion by Prochure** [PKU-Alignment/omnisafe#317] [Q&A] [answered]
Hello, does this box support multi-agent reinforcement learning?

**Comment by Gaiejj** [comment, marked as answer]
Currently, OmniSafe does not support multi-agent safe reinforcement learning algorithms. This will become one of our TODOs. If you have algorithms you hope to implement or other requirements, please feel free to continue leaving messages.
